python-avd/pyavd/_eos_designs/schema/schema_fragments/defs_node_type.schema.yml

```diff
@@ -1383,6 +1383,14 @@ $defs:
                   type: str
                   description: |
                     L3 interface profile name. Profile defined under `l3_interface_profiles`.
+          l3_port_channels:
+            type: list
+            $ref: "eos_designs#/$defs/node_type_l3_port_channels"
+            documentation_options:
+              table: node-type-l3-port-channels-configuration
+            description: |-
+              L3 Port-Channel interfaces to configure on the node.
+              Used to define the node for WAN interfaces when `wan_carrier` is set.
```

Comment:
I don't understand this part. @gmuloc

Reply:
Hi @ClausHolbechArista 
I was mimicing description as being used for "node-type-l3-interfaces" in schema.
L3 Port-Channel interface would be used to define a wan-facing interface when "wan_carrier" attribute is set.
Do you suggest anything else here?

Reply:
Have addressed the description in schema for both node types -  `l3_interfaces` and `l3_port_channels`